tests/testRandom1.py

```diff
@@ -91,8 +91,8 @@ def testPolicy(self):
         pol = pexPolicy.Policy()
         seed = getSeed()
         pol.set("rngSeed", str(seed))
-        pol.set("rngAlgorithm", afwMath.Random.getAlgorithmNames()[afwMath.Random.RANLXD2])
-        r1 = afwMath.Random(afwMath.Random.RANLXD2, seed)
+        pol.set("rngAlgorithm", afwMath.Random.getAlgorithmNames()[int(afwMath.Random.Algorithm.RANLXD2)])
+        r1 = afwMath.Random(afwMath.Random.Algorithm.RANLXD2, seed)
```

Comment:
Yes to the cast, but no to the scope (which I now changed back). In Swig enums are really just ints whereas in pybind11 they are actual enums which means casts are sometimes needed. But on the plus side, two different enum types can now no longer accidentally be exchanged in function calls. Pybind11 is safer.